CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 - Fixed choppiness of the graphs generated in the export modal
 - Fixed matrix transformation and intersection checking bug when sending graph data to front end
 - Fixed inverse matrix storage from column-major to row-major order
+- Fixed SVG Graph Creation (translation issue)
```

Comment:
This entry can be improved in a few ways:

1. Discuss the specific user-facing part of the application (it's not just "svg graph creation")
2. The issue is more general than just translations
3. In terms of writing style, the entry should be written as a standard English sentence (though without a period), and in particular pay attention to capitalization and grammar